### DISCUSSION TITLE
Collaboration with translation

### DISCUSSION THREAD

**Discussion by fattain-naime** [d4rken-org/sdmaid-se#619] [General]
I'm interested in collaboration. 

I want to translate the Bangla language. I'm from Bangladesh and our mother language is bangla. I can translate Bangla very well.

I have purchased a lifetime membership of the sd maid se. 

How do i start the transition?

Thank you

**Comment by d4rken** [comment]
Translations are managed here, just sign up and start translating :grin:
https://crowdin.com/project/sdmaid-se

Thanks for your help! I'm sure many users will be happy about seeing the app in their native language.

**Reply by fattain-naime** [reply]
Thank you. i will complete all bangla Translation.
